src/main/java/com/google/sps/DataServlet.java

```diff
@@ -103,7 +104,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       // InputStreamReader(getServletContext().getResourceAsStream("/WEB-INF/mutations.textproto"));
       // MutationList.Builder mutBuilder = MutationList.newBuilder();
       // TextFormat.merge(mutReader, mutBuilder);
-      // List<Mutation> mutList = mutBuilder.build().getMutationList();
```

Comment:
Merge with master, I think this code was uncommented.

Reply:
fixed!